### PR TITLE
Added a comment to prevent people from manually updating the changelog

### PR DESCRIPTION
**Pull request by wouterj**
For instance, to prevent things like https://github.com/symfony/symfony-docs/pull/7132

### CODE REVIEW

File: changelog.rst [hunk 1]
```diff
@@ -1,5 +1,9 @@
 .. index::
     single: CHANGELOG
+    
+.. !! CAUTION !!
+   This file is automatically generated each month. Do not add new changelog
```
Comment (javiereguiluz) [inline changelog.rst]: I'd remove `each month` to not constraint ourselves and be able to change the frequency as needed.

File: changelog.rst [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+   items when preparing a pull requests.
```
Comment (javiereguiluz) [inline changelog.rst]: `a pull requests.` -> `a pull request.`

**Review by javiereguiluz — APPROVED** [review]
👍

**Comment by xabbuh** [comment]
👍

**Comment by xabbuh** [comment]
Thank you Wouter.

**Pull request merged**